src/providers/liveupdates/BasicPubSubManager.hpp

```diff
@@ -337,12 +337,15 @@ class BasicPubSubManager
     WebsocketContextPtr onTLSInit(const websocketpp::connection_hdl & /*hdl*/)
     {
         WebsocketContextPtr ctx(
-            new boost::asio::ssl::context(boost::asio::ssl::context::tlsv12));
+            new boost::asio::ssl::context(boost::asio::ssl::context::tlsv13));
```

Comment:
Should probably be `tls_client` to support both v1.2 and v1.3.

Reply:
If possible this would be preferred yeah. We know the issues of TLS v1.3-only support :disappointed:

Reply:
confirmed `tls_client` also works for bttv & 7tv, thanks!